modules/nf-core/skani/search/meta.yml

```
@@ -0,0 +1,54 @@
+name: "skani_search"
+description: Memory-efficient ANI database queries with skani.
+keywords:
+  - skani
```

```suggestion
  - ani
  - skani
```
and in all other modules